### DISCUSSION TITLE
How to run conductor and community extension

### DISCUSSION THREAD

**Discussion by astelmashenko** [Netflix/conductor#2976] [Q&A]
I'm looking 3.8.x release, and it's great decision to me. What I can not get is how to run conductor now? I'm trying to add community extension to classpath but it does not work. I put `conductor-nats-3.8.0.jar` into lib folder and run it like below
```
export config_file=~/configs/local.properties
java -Dloader.path="lib/" -DCONDUCTOR_CONFIG_FILE=$config_file -jar conductor-server-3.8.0-boot.jar
```

The only way I was able to run conductor with community lib was I added 
```
implementation 'com.netflix.conductor:conductor-nats:3.8.0'
```
directly into `conductor/server/build.gradle` and rebuilt the project. Is it the right way? Can I just download library and make it working without recompiling?

**Comment by v1r3n** [comment]
Hi @astelmashenko you can use the -classpath option to collect all the jars instead of -jar option.

**Reply by astelmashenko** [reply]
Hi @v1r3n , if I remove -jar java will not know what to run. If you know how to run conductor-server AND one or two community extensions  please write this command. I tried what you suggested:
`java -DCONDUCTOR_CONFIG_FILE=$config_file -classpath conductor-nats-3.8.0.jar:conductor-server-3.8.0-boot.jar`
And it is not valid command, jvm needs `-jar` to be specified.
The problem it is `conductor-server-3.8.0-boot.jar` is a boot jar, it has all libraries inside, and it does not see `conductor-nats-3.8.0.jar`
